### DISCUSSION TITLE
Webui

### DISCUSSION THREAD

**Discussion by Thelvesa** [haugene/docker-transmission-openvpn#1909] [Q&A] [answered]
I did not put this under issues;

Can't figure out how to change the webui

Im using the docker on synology, everything is working as intented, i just don't like the webui ( not enough info on webui )

Have read the doc, and it's explained there, but i don't have those values in my container.

Thank you in advance.

**Comment by pkishino** [comment]
Hey, good on you for not creating an issue on this.
if you don't have the value, which would be a bit strange, you can just add the env variable.
Are you using portainer or such to manage to containers?

**Reply by Thelvesa** [reply]
I have portainer, this is my env
![open](https://user-images.githubusercontent.com/26679494/129961182-35d8f0c6-ec50-4dd0-923d-e05a97cd57f5.jpg)

**Reply by pkishino** [reply]
Are those all?
it's missing
LOCAL_NETWORK
which is required to work properly..
anyway,
just re-create the container and LOCAL_NETWORK and 
TRANSMISSION_WEB_UI
then you can specify the UI

**Comment by Thelvesa** [comment]
it's working without the local_network.

Did the re-create contrainer from portainer. Didn't change anythinng inside enviroment

**Reply by pkishino** [reply]
Cleared your cache? You shouldn’t be able to access the ui without LOCAL_NETWORK set.. I’d have to check if it uses default and you happen to use the same subnet, but you really should add that

**Reply by Thelvesa** [reply]
How do i clear the cache?

**Reply by pkishino** [reply]
depends on the browser,
in chrome,try cmd+shift+r to refresh and clear..

**Reply by Thelvesa** [reply]
didn't change anything, not to rush or anything, any idea when 4.0 will come? Might be better to wait until then and do a new install

**Comment by pkishino** [comment, marked as answer]
It will come when it comes..
you should check the log to make sure the env variable is read properly and
it should state which UI it will use..

**Reply by Thelvesa** [reply]
will give it a try, will respond to this tread if i meet another wall, ty for your help
